source/includes/gcp/_target_proxies.md

```diff
@@ -84,13 +89,62 @@ Retrieve a target proxy in an [environment](#administration-environments).
 | -------------------------------- | ------ |
 | `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
 | `id`<br/>*UUID*                  | The id of the backend service. |
-| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies.|
+| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies. |
 | `name`<br/>*string*              | Name of the resource. |
 | `selfLink`<br/>*string*          | Server-defined URL for the resource. |
 | `shortUrlMap`<br/>*string*       | A short version of the URL map. |
-| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP|
+| `shortCertificates`<br/>*string* | A short version of the SSL certificates. |
```

Comment:
'short version' is not that much descriptive whereas 'name of URL map' will make it more clear to the reader as per me. 
The same goes for type.

Reply:
How about "A short version of the URL map name"? This would follow the same style as the rest of the doc.

Reply:
ae81aa9d03315427864b7953c7932bce5c8f5420